### ISSUE TITLE
[sirius] WARNING in input/input_keyword_types.F:284 :: keyword XC_FUNCTIONALS

### ISSUE THREAD

**Issue by dev-zero** [cp2k/cp2k#317]
When building CP2K with support for SIRIUS, I get the following when running `cp2k --help`:

```console
$ cp2k.popt --help

 cp2k.popt [-c|--check] [-e|--echo] [-h|--help] [--html-manual]
           [-i] <input_file>
           [-mpi-mapping|--mpi-mapping] <method>
           [-o] <output_file>
           [-r|-run] [--xml]

 starts the CP2K program, see <https://www.cp2k.org/>

 The easiest way is cp2k.popt <input_file>

 The following options can be used:

  -i <input_file>   : provides an input file name, if it is the last
                      argument, the -i flag is not needed
  -o <output_file>  : provides an output file name [default: screen]

 These switches skip the simulation, unless [-r|-run] is specified:

  --check, -c       : performs a syntax check of the <input_file>
  --echo, -e        : echos the <input_file>, and make all defaults explicit
                      The input is also checked, but only a failure is reported
  --help, -h        : writes this message
  --html-manual     : writes a HTML reference manual of the CP2K input
                      in the current directory. The file index.html is a good
                      starting point for browsing
  --license         : prints the CP2K license
  --mpi-mapping     : applies a given MPI reordering to CP2K
  --run, -r         : forces a CP2K run regardless of other specified flags
  --version, -v     : prints the CP2K version and the revision number
  --xml             : dumps the whole CP2K input structure as a XML file
                      xml2htm generates a HTML manual from this XML file
  

 *** WARNING in input/input_keyword_types.F:284 :: keyword XC_FUNCTIONALS ***
 *** assumed undefined type by default                                    ***
```

@mtaillefumier unfortunately this only happens when building CP2K with SIRIUS support.

**Comment by dev-zero** [comment]
this is with SIRIUS v6.1.5 and libvdwxc v0.4.0 and CP2K  9b85492

**Comment by mtaillefumier** [comment]
I will have to find a way to fix it. XC_FUNCTIONALS should take a list of strings but I do not know how to do that with the cp2k parser. It has no implication though since I use the cp2k XC_FUNCTIONAL information instead.

**Comment by mkrack** [comment]
@mtaillefumier In the section &GENPOT for [PARAMETERS](https://manual.cp2k.org/trunk/CP2K_INPUT/FORCE_EVAL/MM/FORCEFIELD/NONBONDED/GENPOT.html#list_PARAMETERS) a list of words (strings) is read/parsed. See line 1354 in input_cp2k_mm.F
HTH

**Comment by stanos4** [comment]
Hi, is this problem solved already? (I want to ask before doing a very annoying process of update on offline cluster...). 

This error breaks codes using the cp2k_shell server, since those expect to receive only " * READY" line upon startup, such as in ASE CP2K calculator.

Thank you!

**Comment by mtaillefumier** [comment]
fixed by this pull request. https://github.com/cp2k/cp2k/pull/877

**Comment by oschuett** [comment]
Fixed by #877 and prevented from reapparing via https://github.com/cp2k/cp2k/commit/7f99a85e0841a1d726782a1041832aac3b8130c4.
